docs/src/appendix/forge-library/precalculate_address.md

```diff
@@ -0,0 +1,37 @@
+# `precalculate_address`
+
+> ⚠️ **Warning**
+>
+> Precalculated address is only correct for the very next deploy call. It will be different for all deploy calls after the first one after calculating the address.
```

Comment:
```suggestion
> Precalculated address is only correct for the very next [`deploy`](./deploy.md) call. It will be different for all `deploy` calls after the first one after calculating the address.
```